magento/magento2
# Auto select https if port is 443 and not protocol is detected

Summary:
Fixes solution 1 of Issue 37490 by defaulting to protocol https if no protocol is given in the hostname and the port given is 443

Issues:
https://github.com/magento/magento2/issues/37490

Testing:
Tested on 2.4-develop on AWS EC2 instance with AWS Opensearch domain running version 1.3



### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Partially fixes magento/magento2#37490

### Manual testing scenarios (*)

1. Create a fresh instance, in opensearch params use port 443 and don't provide a protocol in hostname, the resulting hostname should have protocol https instead of the original default to http

### Questions or comments


### Contribution checklist (*)
